fabric-datafixer-v1/src/main/java/net/fabricmc/fabric/impl/datafixer/FabricDataFixerImpl.java

```diff
@@ -187,7 +187,7 @@ public void registerTypes(Schema schema, Map<String, Supplier<TypeTemplate>> ent
 	/**
 	 * Represents Minecraft's Built in DataFixer.
 	 */
-	public static class MCDFU {
+	public static final class MCDFU {
```

Comment:
`McDfu`

Reply:
Why not McDataFixerUpper or MinecraftDataFixerUpper or VanillaDataFixerUpper or VanillaDataFixer or MinecraftDataFixer ?